pkg/virt-handler/container-disk/mount.go

```diff
@@ -256,59 +254,45 @@ func (m *mounter) MountAndVerify(vmi *v1.VirtualMachineInstance) (map[string]*co
 	if len(record.MountTargetEntries) > 0 {
 		err := m.setMountTargetRecord(vmi, &record)
 		if err != nil {
-			return nil, err
+			return err
 		}
 	}
 
-	vmiRes, err := m.podIsolationDetector.Detect(vmi)
-	if err != nil {
-		return nil, fmt.Errorf("failed to detect VMI pod: %v", err)
-	}
-
 	for i, volume := range vmi.Spec.Volumes {
 		if volume.ContainerDisk != nil {
 			diskTargetDir, err := containerdisk.GetDiskTargetDirFromHostView(vmi)
 			if err != nil {
-				return nil, err
+				return err
 			}
 			diskName := containerdisk.GetDiskTargetName(i)
 			targetFile, err := safepath.JoinNoFollow(diskTargetDir, diskName)
 			if err != nil {
-				return nil, err
+				return err
 			}
 
 			if isMounted, err := isolation.IsMounted(targetFile); err != nil {
-				return nil, fmt.Errorf("failed to determine if %s is already mounted: %v", targetFile, err)
+				return fmt.Errorf("failed to determine if %s is already mounted: %v", targetFile, err)
 			} else if !isMounted {
 
 				sourceFile, err := m.getContainerDiskPath(vmi, &volume, i)
 				if err != nil {
-					return nil, fmt.Errorf("failed to find a sourceFile in containerDisk %v: %v", volume.Name, err)
+					return fmt.Errorf("failed to find a sourceFile in containerDisk %v: %v", volume.Name, err)
 				}
 
 				log.DefaultLogger().Object(vmi).Infof("Bind mounting container disk at %s to %s", sourceFile, targetFile)
 				out, err := virt_chroot.MountChroot(sourceFile, targetFile, true).CombinedOutput()
 				if err != nil {
-					return nil, fmt.Errorf("failed to bindmount containerDisk %v: %v : %v", volume.Name, string(out), err)
+					return fmt.Errorf("failed to bindmount containerDisk %v: %v : %v", volume.Name, string(out), err)
 				}
 			}
-
-			imageInfo, err := isolation.GetImageInfo(containerdisk.GetDiskTargetPathFromLauncherView(i), vmiRes, m.clusterConfig.GetDiskVerification())
```

Comment:
@vladikr i missed the fact that we verify the disk memory is not higher than `m.clusterConfig.GetDiskVerification().MemoryLimit.Value())`

this is why `pull-kubevirt-e2e-k8s-1.32-sig-compute-serial` keep failing, do you think we can move this check to the launcher pods :\?